docs/reference/operator_parameters.md

```diff
@@ -172,6 +172,11 @@ configuration they are grouped under the `kubernetes` key.
   list of `name:value` pairs for additional labels assigned to the cluster
   objects. The default is `application:spilo`.
 
+* **inherited_labels**
+  list of labels that can be inherited from the cluster manifest, and added to
```

Comment:
can you please briefly mention in the cluster manifest that  such 3d party labels are essential part of the PG cluster configuration but are stored outside of the cluster manifest ?

Reply:
Your formulation is totally fine. We indeed use somewhat arbitrarily the term "cluster manifest" to refer to the Postgres manifest.

I just meant that there should be a note in the [cluster manifest reference](https://github.com/zalando-incubator/postgres-operator/blob/master/docs/reference/cluster_manifest.md) that there is  the `inherited_labels` conf option which values vary among PG clusters but is nevertheless specified outside the Postgres manifest.

Right now the settings specified in the operator conf are the same across all PG clusters; setting specific to the PG cluster are in their individual manifests. Your proposed change sits in between in a sense that it is configured at the operator level but the actual values vary from a PG DB to a PG DB.